code/__DEFINES/mobs.dm

```diff
@@ -431,13 +431,15 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define CASTE_CAN_VENT_CRAWL		(1<<1)
 #define CASTE_CAN_BE_QUEEN_HEALED	(1<<2)
 #define CASTE_CAN_BE_GIVEN_PLASMA	(1<<3)
-#define CASTE_INNATE_HEALING		(1<<4)
+#define CASTE_INNATE_HEALING		(1<<4) // Xenomorphs suffer no heal penalty while on weeds
 #define CASTE_FIRE_IMMUNE			(1<<5)
 #define CASTE_EVOLUTION_ALLOWED		(1<<6)
 #define CASTE_IS_INTELLIGENT		(1<<7) // A hive leader or able to use more human controls
 #define CASTE_DECAY_PROOF			(1<<8)
 #define CASTE_CAN_BE_LEADER			(1<<9)
 #define CASTE_HIDE_IN_STATUS		(1<<10)
+#define CASTE_NO_STAND_HEAL_PENALTY	(1<<11) // Xenomorphs don't suffer a penalty to healing when not resting
```

Comment:
Bitflags should be positive rather than negative, since then you may be basically doing if `!isn't_something`